example/methods_directory/physical/psp.json

```diff
@@ -1,49 +1,52 @@
 {
-    "name": "PAW Pseudopotential method (VASP)",
-    "path": "/qm/wf/psp/paw",
+    "slug": "pseudopotential",
     "categories": {
         "tier1": "qm",
         "tier2": "wf",
         "type": "psp",
         "subtype": "paw"
     },
     "data": {
-        "pseudo": [
+        "element": "Ru",
```

Comment:
"data" has to contain an array. We can replace "pseudo" with "metaProperties".

Reply:
Ok, need to refactor in `web-app/src/application/imports/properties/server/startup.js`